extension/data/frame/frame.css

```diff
@@ -0,0 +1,33 @@
+body {
+  margin-top: 1px;
+  font-family: "Lucida Grande", "Segoe UI", Arial, sans-serif;
+  font-size: 11px;
+  color: inherit;
+  text-align: right;
```

Comment:
As a MemChaser user I think it should have the default alignment. :)

There are too much button near there: menu button, logging button, close toolbar button...
If somebody close the toolbar accidentally she should turn it back manually with left click.